python/lsst/ts/electrometer/csc.py

```diff
@@ -312,7 +312,7 @@ async def do_startScan(self, data):
         )
         try:
             await self.report_detailed_state(DetailedState.MANUALREADINGSTATE)
-            await self.controller.start_scan()
+            await self.controller.start_scan(group_id=data.groupId)
```

Comment:
you may want to make this backward compatible. At least until we release the new version of the xml. 

you can do that with

```
await self.controller.start_scan(group_id=getattr(data, "groupId", None))
```

This will make sure that it passes `None` for the current version of the CSC.